cli/src/commands/sidekick.rs

```diff
@@ -446,4 +450,15 @@
 
 		Ok(())
 	}
+
+	pub fn get_cli_version(&self) -> Result<()> {
+		struct_fmt::print(
+			&Format::Json,
+			&SideKickResponse::Ok(json!({
+				"version": concat!("v", env!("VERGEN_BUILD_SEMVER"))
+			})),
+		)?;
```

Comment:
I'm going to swap out how this works downstack so that it's a lot cleaner, but I'll swap that over once I do it :)